packages/cli-kit/assets/cli-ruby/lib/shopify_cli/theme/dev_server/hot_reload/script_injector.rb

```diff
@@ -25,7 +25,7 @@ def inject(body:, dir:, mode:)
               "</script>",
             ].join("\n")
 
-            body = body.join.gsub("</body>", "#{hot_reload_script}\n</body>")
+            body = body.join.sub("</body>", "#{hot_reload_script}\n</body>")
```

Comment:
As a valid HTML document should contain only **1** `<body>`, I guess it’s okay if we only replace it **once**, right? :D